en/docs/Administer/ProductSecurity/enabling-java-security-manager.md

```diff
@@ -1,157 +1,159 @@
-# admin\_Enabling Java Security Manager
+# Enabling Java Security Manager
 
 !!! warning
-**Please note that this document is currently under review. Currently, WSO2 does not recommend Java security manager for production environments due to some known issues.**
+    **Please note that this document is currently under review. Currently, WSO2 does not recommend Java security manager for production environments due to some known issues.**
 
 
 The Java Security Manager is used to define various security policies that prevent untrusted code from manipulating your system. Enabling the Java Security Manager for WSO2 products activates the Java permissions that are in the `<PRODUCT_HOME>/repository/conf/sec.policy` file. You modify this file to change the Java security permissions as required.
 
 !!! info
-**Before you begin**
+    **Before you begin**
 
--   Ensure that you have Java 1.8 installed.
--   Note that you need to use a keystore for signing JARs using the Java security manager. In this example, you will be using the default keystore in your WSO2 product ( `wso2carbon.jks` ). You can read about the recommendations for using keystores from [here](https://docs.wso2.com/display/ADMIN44x/Using+Asymmetric+Encryption#UsingAsymmetricEncryption-RecommendationsforsettingupkeystoresinWSO2products) .
+    * Ensure that you have Java 1.8 or Java 11 installed.
+    * Note that you need to use a keystore for signing JARs using the Java security manager. In this example, you will be using the default keystore in your WSO2 product ( `wso2carbon.jks` ). You can read about the recommendations for using keystores from [here](https://docs.wso2.com/display/ADMIN44x/Using+Asymmetric+Encryption#UsingAsymmetricEncryption-RecommendationsforsettingupkeystoresinWSO2products).
 
 
 The steps below show how to enable the Java Security Manager for WSO2 products.
 
 1.  Download the WSO2 product to any location (e.g., `<HOME>/user/<product-pack>` folder).
 
-2.  To sign the JARs in your product, you need a key. You can generate a new keystore (with a new key) by executing the keytool command given below. Note that the new keystore is created in the directory from which you execute the keytool command.
+2. Run the startup script in the `<PRODUCT_HOME>/bin` folder. For Linux, it is `wso2server.sh`. Once the product is up and running, shutdown the server.
```

Comment:
<PRODUCT_HOME> should be changed to <API-M_HOME>

Reply:
Fixed in f906106